client_receiver.py

```diff
@@ -0,0 +1,22 @@
+"""
+Client receive messages from server and print them
+"""
+
+import asyncio
```

Comment:
V requirements neni informace o tom, ze se musi nainstalovat nove knihovny - to je potreba pridat.

Reply:
tak mi to schval a já si udělám issue na dopsání requirements a dohledám si co všechno je potřeba :-)  :-D

Reply:
v requirements doplněno